Lib/test/test_inspect/test_inspect.py

```diff
@@ -2992,6 +2992,14 @@ def test2(pod=42, /):
         with self.assertRaisesRegex(ValueError, 'follows default argument'):
             S((pkd, pk))
 
+        second_args = args.replace(name="second_args")
+        with self.assertRaisesRegex(ValueError, 'more than one var-positional parameter'):
+            S((args, second_args))
```

Comment:
Add also a test for the case when there are other parameters (keyword-only or var-keyword) between two var-positional parameters.

Reply:
In this case we will get error: `ValueError: wrong parameter order: keyword-only parameter before variadic positional parameter`. Should we catch it in the test?

My test:`S((args, ko, second_args))`

Reply:
Technically, there are two errors: "keyword-only parameter before variadic positional parameter" and "more than one variadic positional parameter". Which of them are preferable to report? I think the latter, therefore we should change the order of checks. And tests are needed to ensure that the correct error message is used.

Reply:
In that case we all need the variables seen_var_positional and seen_var_keyword, to check that these parameters have not occurred before even if the parameter order is wrong, right?
`if kind == top_kind and kind in (_VAR_POSITIONAL, _VAR_KEYWORD)` - this check is designed for the correct order of arguments

Reply:
Hmm, yes, you are right. So you need to simply move the new checks up. I still suggest to use `kind.description` to format the error message.

You can unify the code for var-positional and var-keyword if use a set instead of two boolean variables.
```py
if kind in (_VAR_POSITIONAL, _VAR_KEYWORD):
    if kind in seen_var_parameters:
        raise ...
    seen_var_parameters.add(kind)
```